### ISSUE TITLE
Does this class still work in iOS 6?

### ISSUE THREAD

**Issue by davidolesch** [gohypergiant/BPXLUUIDHandler#7]
I tried CocoaPods install and cloning the repo but I always get these missing reference linker errors.

![Screen Shot 2013-04-23 at 3 18 50 PM](https://f.cloud.github.com/assets/1456095/416642/0f815720-ac53-11e2-80c7-f324223fe398.png)

**Comment by orta** [comment]
You have to include the Security framework

**Comment by davidolesch** [comment]
Thanks @orta. That was my problem. Shouldn't the README mention that the security framework is required?

**Comment by orta** [comment]
Nope, you have a PR that want's to fix that: https://github.com/blackpixel/BPXLUUIDHandler/pull/6

**Comment by bar2** [comment]
I see that 36 digit long keys are automatically converted into 40 digit long keys as of iOS 6. Is it safe to use BPXUUIDHandler for iOS 6?

**Comment by ngreenstein** [comment]
Hi David,

I can't reproduce this; `[BPXLUUIDHandler UUID]` doesn't produce any errors with or without `Security.framework`. What version of Xcode are you using and what version of iOS are you targeting?

Thanks
